### DISCUSSION TITLE
script模式能否访问网络？

### DISCUSSION THREAD

**Discussion by linesoft2** [zzzgydi/clash-verge#477] [General]
🥺

**Comment by zzzgydi** [comment]
你这里的script模式是指的clash p核的script还是，script profile呢

**Reply by linesoft2** [reply]
script profile

**Reply by zzzgydi** [reply]
是有啥场景么

**Reply by Maverick3a1** [reply]
> 是有啥场景么

这是cfw的文档。
![image](https://github.com/zzzgydi/clash-verge/assets/25895481/0a8438d3-ab92-494e-99bf-9649ecb38058)

Proxy Groups的头部实际上可以做成一个类似于Surge Panel的功能，把一些关注的信息做成Proxy写在上面，然后Profile一个小时自动更新一次，上面的内容就自动更新了。

**Reply by zzzgydi** [reply]
> Proxy Groups的头部实际上可以做成一个类似于Surge Panel的功能

类似Surge Panel的功能是啥，还没用过Surge

**Reply by Maverick3a1** [reply]
就是一个信息面板，上面可以自定义内容，方便查看。
